clang/lib/DPCT/CallExprRewriter.cpp

```diff
@@ -1247,6 +1247,31 @@ makeCallArgCreator(std::string Str) {
   return [=](const CallExpr *C) -> const StringRef { return StringRef(Str); };
 }
 
+std::function<std::string(const CallExpr *)>
+makeCallArgCreatorFromTemplateArg(unsigned Idx) {
```

Comment:
can reuse lambda in makeTemplatedCalleeCreator

Reply:
done